build/ci/unit-tests.yml

```diff
@@ -61,3 +61,21 @@ jobs:
     pool: 
       name: NetCorePublic-Pool
       queue: BuildPool.Windows.Amd64.VS2019.Pre.ES.Open
+
+- job: RichNav
+  pool:
+    name: NetCorePublic-Pool
+    queue: BuildPool.Windows.10.Amd64.VS2019.Pre.Open
+  variables:
+    _configuration: Debug
+  timeoutInMinutes: 240
+  steps:
+    - script: $(Build.SourcesDirectory)\build.cmd /build /ci /configuration $(_configuration)
```

Comment:
Does it actually use binaries or just source code? Above is building the tree.

Reply:
It needs to be able to instantiate a `MSBuildWorkspace` (or the moral equivalent) so I believe a restore is all that is necessary.

Reply:
The indexer does not use the binaries

Reply:
Okay so we need a restore concept in the build.

Reply:
I'm working on it.

Reply:
Only after this: https://github.com/dotnet/project-system/pull/5327, you can now do a restore only by doing the following:

```
build /no-build /no-test /no-deploy
```